### ISSUE TITLE
More Status Icons

### ISSUE THREAD

**Issue by corinnaj** [cmfcmf/roc-and-roll#85]
- [ ] Flying
- [ ] Guided

**Comment by cmfcmf** [comment]
We could also consider to introduce a "height" field or similar for tokens that can be used to indicate at which height someone is flying 🤔

**Comment by tom95** [comment]
I agree, altitude or z come to mind as names. Could be visualized through clever use of shadows + a label next do the token that indicates the number in ft.
